oneview_redfish_toolkit/client_session.py

```diff
@@ -44,6 +49,49 @@ def init_map_clients():
     globals()['map_clients'] = OrderedDict()
 
 
+def init_gc_for_expired_sessions():
+    gc_thread = threading.Thread(target=_gc_for_expired_sessions)
+    gc_thread.daemon = True
+    gc_thread.start()
+
+
+def _gc_for_expired_sessions():
+    time.sleep(GC_FREQUENCY_IN_SEC)
+    logging.info('Verifying expired sessions...')
+
+    tokens_to_remove = []
+    map_items = _get_map_clients().items()
+    for token, dict_by_token in map_items:
+        ov_clients = iter(dict_by_token['client_ov_by_ip'].values())
+        ov_client = next(ov_clients)  # ov_clients can be in any order
+        ov_session_id = ov_client.connection.get_session_id()
```

Comment:
It's checking for just one OneView for each token session right? I'm thinking on multiple OneView scenario where one of them have different settings for expiration time. Do you think that we need to cover this scenario @adriane-cardozo ?

Reply:
I imaginate that oneviews should be configured with same expiration time.
I think checking just one client just be sufficient, because all should expire in similar time.

Reply:
Yes, let's assume all OV have the same expiration time for now.